examples/deployment/scripts/multisig/multisig_goal_sc.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Description:
+ * This file demonstrates the example to
+   - load a signed lsig from assets and use that lsig to validate transactions
+*/
+const { transferAlgo } = require("../transfer/common");
+const { transferMicroAlgos, getmultisigAddress } = require("algob");
+
+async function run(runtimeEnv, deployer) {
+  const goldOwnerAccount = deployer.accountsByName.get("gold-owner-account");
+  const johnAccount = deployer.accountsByName.get("john-account");
+  const bobAccount = deployer.accountsByName.get("bob-account");
+
+  /*
+    This example demonstrates a transaction validated by a smart contract signed by a multisig account with threshold of 2
+    User saves the signed logic signature file in assets/ which is extracted and used here.
+  */
+  
+  //Generate multi signature account hash 
+  const addrs =  [goldOwnerAccount.addr, johnAccount.addr, bobAccount.addr]  // you can replace these addresses with your custom addrs for multisig account.
+  const [mparams, kmdMsigAddress] = getmultisigAddress(1, 2, addrs);   // passing (version, threshold, address list)
```

Comment:
are these accounts already in goal ? or how will the user create them ?

Reply:
These are accounts created by us to demonstrate / run the example which contains the signed mlsig by 2 of these accounts (gold and john). If user loads a custom signed lsig, he will need to update these addresses with his own. This is just an example.

Reply:
what i am trying to say is " user should be able to create these accounts and test them "
- " this should not be environment dependent"
- "you should mention in readme how the user will create and use accounts"